README.md

```diff
@@ -2,14 +2,30 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  
+  1. Real world conditions/ algorithm implementation: Asymptotic analysis focuses on the growth rate of an algorithm and its performance as the input size increases. However it normally does not account for pratical implementation details, while it does consider worst, best, and average case scenarios, it still assumes that the algorithm is implemented according to its theoretical design. So it does not factor in potential inefficiencies thay may arise in a real world scenario. For example, an algorithm can be influenced by factors like cache behavior or actual input distribution which are often not captured in asymptotic analysis. Which could cause a theoretically worse asymptotic complexity to outperform another with a much better complexity due to optimizations or favorable conditions.
```

Comment:
Asymptotic analysis does not assume that the algorithm is implemented in a certain way.